store_data.py

```diff
@@ -0,0 +1,355 @@
+                            "time_for_connect_sd_percent FLOAT UNSIGNED,"
+                            "req_per_sec_max INTEGER UNSIGNED,"
+                            "req_per_sec_min INTEGER UNSIGNED,"
+                            "req_per_sec_mean INTEGER UNSIGNED,"
+                            "req_per_sec_sd INTEGER UNSIGNED,"
+                            "req_per_sec_sd_percent FLOAT UNSIGNED,"
+                            "total_req_per_sec DOUBLE UNSIGNED,"
+                            "time_for_request_max INTEGER UNSIGNED,"
+                            "time_for_request_min INTEGER UNSIGNED,"
+                            "time_for_request_mean INTEGER UNSIGNED,"
+                            "time_for_request_sd INTEGER UNSIGNED,"
+                            "time_for_request_sd_percent FLOAT UNSIGNED,"
+                            "status_codes_2xx BIGINT UNSIGNED,"
+                            "status_codes_3xx BIGINT UNSIGNED,"
+                            "status_codes_4xx BIGINT UNSIGNED,"
+                            "status_codes_5xx BIGINT UNSIGNED,"
+                            "traffic_total_bytes BIGINT UNSIGNED,"
+                            "traffic_total_data_bytes BIGINT UNSIGNED,"
+                            "traffic_total_headers_bytes BIGINT UNSIGNED,"
+                            "traffic_total_savings FLOAT UNSIGNED,"
+                            "PRIMARY KEY (id)"
+                            ")")
+  db_utils.ExecuteAndReturnResult(connection,
+                                  create_table_statement.format(table_name))
+
+
+def GetMicrosecondData(data):
+  """The function accepts a dictionary and returns microsecond data inside it.
+
+  Based on the unit (s, ms, us) inside the dictionary, the function converts to
+  microsecond value.
+  Args:
+    data: data is a dictionary of the format: {data: <val>, unit: <unit>}.
+  Returns:
+    A float value on the microsecond unit.
+  """
+  assert data["unit"] == "us" or data["unit"] == "ms" or data["unit"] == "s"
+
+  if data["unit"] == "us":
+    return float(data["data"])
+  elif data["unit"] == "ms":
+    return float(data["data"]) * 1000
+  elif data["unit"] == "s":
+    return float(data["data"]) * 1000000
+
+
+def GetByteData(data):
+  """The function accepts a dictionary and returns Bytes value inside it.
+
+  Based on the unit (B, MB, KB), the function converts to Bytes value. It also
+  works for B/s, MB/s, etc.
+  Args:
+    data: data is a dictionary of the format: {data: <val>, unit: <unit>}.
+  Returns:
+    A float value of the Bytes unit.
+  """
+  assert (data["unit"].startswith("B") or data["unit"].startswith("K") or
+          data["unit"].startswith("M"))
+
+  if data["unit"].startswith("B"):
+    return int(data["data"])
+  elif data["unit"].startswith("K"):
+    return int(float(data["data"]) * 1024)
+  elif data["unit"].startswith("M"):
+    return int(float(data["data"]) * 1024 * 1024)
+
+
+def JSONToInsertCommand(table, **kwargs):
```

Comment:
This should have a more descriptive name - it's not working with JSON at all (the caller is). E.g. `InsertCommandFromArgs`.